manifest.json

```diff
@@ -23,21 +23,23 @@
 	},
 
 	"background": {
-		"scripts": ["background.js"]
+		"scripts": ["engine.js", "background.js"]
 	},
 
 	"content_scripts": [
 		{
 			"matches": ["https://*/*", "http://*/*"],
-			"js": ["inject-referer.js"],
+			"match_about_blank": true,
```

Comment:
Why should the content scripts run in blank frames?

Reply:
A page can embed `<iframe id="myiframe" src="about:blank"></iframe>`, then grab a reference to non-hooked `get referrer` and use that on his own document.
This is to prevent that.

Reply:
Shouldn't such an iframe see the enclosing document URL as its referrer? Can I check the behavior somewhere, or better reproduce it in a test?

Reply:
> Shouldn't such an iframe see the enclosing document URL as its referrer?

Precisely! That check is there so that an iframe see the enclosing document URL as its referrer.

Reply:
That's a mean trick! I'll have to try that myself, and it needs to be in a test! :sweat_smile:

Reply:
Sorry, edited code.

Reply:
I've pushed an update to the tests that checks this issue, see the [`iframe-test` branch](https://github.com/airtower-luna/referer-mod/tree/iframe-test). If you merge that into your branch we can see if the modification works. Thank you for the explanation and example! :slightly_smiling_face:

Reply:
Looks like the tests passed

Reply:
All green, nice!